[debug] [2025-08-25T05:29:52.806Z] ----------------------------------------------------------------------
[debug] [2025-08-25T05:29:52.811Z] Command:       /Users/inderjeet/.nvm/versions/node/v22.12.0/bin/node /Users/inderjeet/.nvm/versions/node/v22.12.0/bin/firebase login
[debug] [2025-08-25T05:29:52.812Z] CLI Version:   14.14.0
[debug] [2025-08-25T05:29:52.812Z] Platform:      darwin
[debug] [2025-08-25T05:29:52.812Z] Node Version:  v22.12.0
[debug] [2025-08-25T05:29:52.812Z] Time:          Mon Aug 25 2025 10:59:52 GMT+0530 (India Standard Time)
[debug] [2025-08-25T05:29:52.812Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-08-25T05:29:52.813Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  The Firebase CLI’s MCP server feature can optionally make use of Gemini in Firebase. Learn more about Gemini in Firebase and how it uses your data: https://firebase.google.com/docs/gemini-in-firebase#how-gemini-in-firebase-uses-your-data 
[debug] [2025-08-25T05:29:53.368Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: unable to get local issuer certificate
[warn] ⚠  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2025-08-25T05:29:53.369Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[info] 
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you. 
[info] 
[info] i  To change your preferences at any time, run `firebase logout` and `firebase login` again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=334798530&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2025-08-25T05:30:39.716Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2025-08-25T05:30:39.717Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2025-08-25T05:30:39.780Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------4f4b6ad91aac7a62117b50f5","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: unable to get local issuer certificate
[debug] [2025-08-25T05:30:39.781Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (/Users/inderjeet/.nvm/versions/node/v22.12.0/lib/node_modules/firebase-tools/lib/apiv2.js:261:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
[error] Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2025-08-25T05:30:39.784Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2025-08-25T05:30:39.785Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"0ecb05a3-a430-407d-9740-089dc04539d0"}
[debug] [2025-08-25T05:30:40.428Z] <<< [apiv2][status] POST https://auth.firebase.tools/attest 200
[debug] [2025-08-25T05:30:40.429Z] <<< [apiv2][body] POST https://auth.firebase.tools/attest {"token":"NsJb6vvVuSPB1BHBfVMEVsLR0OKJDsOeWMq_9LIZ698"}
[info] 
[info] To sign in to the Firebase CLI:
[info] 
[info] 1. Take note of your session ID:
[info] 
[info]    0ECB0
[info] 
[info] 2. Visit the URL below on any device and follow the instructions to get your code:
[info] 
[info]    https://auth.firebase.tools/login?code_challenge=E38N49JQVoJP7ymi3Q7H0W5tMuQxiPWqvG7KIy9tuAo&session=0ecb05a3-a430-407d-9740-089dc04539d0&attest=NsJb6vvVuSPB1BHBfVMEVsLR0OKJDsOeWMq_9LIZ698
[info] 
[info] 3. Paste or enter the authorization code below once you have it:
[info] 
[debug] [2025-08-25T05:30:57.139Z] ExitPromptError: User force closed the prompt with SIGINT
    at Interface.sigint (/Users/inderjeet/.nvm/versions/node/v22.12.0/lib/node_modules/firebase-tools/node_modules/@inquirer/core/dist/commonjs/lib/create-prompt.js:101:37)
    at Interface.emit (node:events:524:28)
    at Interface.emit (node:domain:489:12)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1124:18)
    at ReadStream.onkeypress (node:internal/readline/interface:263:20)
    at ReadStream.emit (node:events:536:35)
    at ReadStream.emit (node:domain:489:12)
    at emitKeys (node:internal/readline/utils:370:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (node:internal/readline/emitKeypressEvents:64:36)
[error] 
[error] Error: An unexpected error has occurred.
